Library/Homebrew/api.rb

```diff
@@ -92,21 +92,39 @@ def self.fetch_json_api_file(endpoint, target:)
 
         retry
       end
+
+      if endpoint.end_with?(".jws.json")
+        success, data = verify_and_parse_jws(json_data)
+        unless success
+          target.unlink
+          odie <<~EOS
+            Failed to verify integrity of #{url} (#{data})."
```

Comment:
```suggestion
            Failed to verify integrity (#{data}) of:
            #{url}
```
the URL is going to be long enough it'll be nicer to stick it on a new line.